website/src/helpers/LoginHelper.jsx

```diff
@@ -35,8 +53,6 @@ export const performAndDispatchLogin = (dispatch) => {
       actionPayload[RC.USER_PHOTO_URL] = result.user.photoURL;
       actionPayload[RC.USER_EMAIL] = result.user.email;
       dispatch(updateUserInfo(actionPayload));
-
-      document.location = "/#/home";
```

Comment:
this will need to be added back if we check based on firebase authentication state, since the userId prop won't be the deciding factor in where we route users